### PR TITLE
patchbay, patchwork: init with appimage-run

### PR DESCRIPTION
**Pull request by astro**
###### Motivation for this change

Template: `standardnotes`

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -4774,6 +4774,10 @@ in
 
   patchage = callPackage ../applications/audio/patchage { };
 
+  patchbay = callPackage ../applications/networking/ssb/patchbay { };
+
+  patchwork = callPackage ../applications/networking/ssb/patchwork { };
```
[the remark this answers is not in the document]
Comment (astro) [reply pkgs/top-level/all-packages.nix]: Ok, renamed to `ssb-patchwork` in that file

**Review by tilpner — CHANGES_REQUESTED** [review]
This is not how appimage-run is meant to be used, it's meant to be a quick escape hatch, like steam-run. appimage-run unnecessarily extracts into the users home directory.

I built an alternative at some point, ~~I'll see if I can find and fix it~~ pr'd as #54693. For the moment, I'd like to find a better option instead of merging this

**Comment by matthiasbeyer** [comment]
We already have patchwork in master.

**Comment by FRidh** [comment]
Patchbay could still actually go in. Even so, I do agree with @tilpner, we should not run an appimage.

**Comment by veprbl** [comment]
@astro Do you plan to finish this?

**Comment by wakest** [comment]
Would love to see patchbay as well as just patchwork, any one working on that? @ehmry?

**Comment by asymmetric** [comment]
Can we close this PR?

**Comment by astro** [comment]
@veprbl Patchwork has landed in nixpkgs regardless of this PR. :fireworks:

**Pull request closed without merging**